src/repository/storage/postgres/orm/sequelize/teams.ts

```diff
@@ -203,6 +217,57 @@ export default class TeamsSequelizeStorage {
     });
   }
 
+  /**
+   * Get note parent structure
+   * @param noteId : the ID of the note.
+   * @param userId : The ID of the user.
+   */
+  public async getAllNoteParents(noteId: NoteInternalId, userId: number): Promise<NoteParentStructure[]> {
+    if (!this.noteModel || !this.noteRelationModel) {
+      throw new DomainError('Note model or Note relation model not initialized');
+    }
```

Comment:
We should not throw `DomainError` inside of the storage, you should use just an `Error`

Reply:
also this error means, that `createAssociationWithNoteRelationsModel` gone wrong, model could be initialized. I would suggest smth like 
```
TeamStorage: Note model is not defined
```
and 
```
TeamStorage: Note relation model is not defined
```